sass/components/_categories.scss

```diff
@@ -0,0 +1,28 @@
+.filters {
```

Comment:
the file is names `categories`, but the class is `filters`. It's a good practice to name the file with the name of the block.